docker/docker-compose.yaml

```diff
@@ -28,9 +29,15 @@ services:
     environment:
       - MAKEPROCS=1
       - RELEASE=1
-    command: bash -cl 'mkdir -p build && cd build && cmake .. && make -j "$${MAKEJOBS}" && make package'
+    command: bash -cl 'mkdir -p build && cd build && cmake .. && make -j "$${MAKEJOBS}" package'
 
   build-snapshot: &build-snapshot
+    <<: *build-setup
+    environment:
+      - MAKEPROCS=1
+    command: bash -cl 'mkdir -p build && cd build && cmake .. && make -j "$${MAKEJOBS}" package'
+
+  test-snapshot: &test-snapshot
```

Comment:
Is there a way to have `build-snapshot` triggered by `test-snapshot`? We don't want a build that's failing tests to be published.

Reply:
If any of the jobs (linux, macos, or tests) fail, nothing is published. An advantage of separating the jobs is knowing WHY the artifacts are not being published. Whether due to a failed build or a failed test.